source/frontend/src/features/contacts/contact/detail/utils.ts

```diff
@@ -28,10 +28,10 @@ export function getContactInfo(
   // Get only the valid types
   let filteredFields = contactMethods.reduce(
     (accumulator: ContactInfoField[], method: IContactMethod) => {
-      if (Object.keys(validTypes).includes(method.contactMethodType.id)) {
+      if (Object.keys(validTypes).includes(method.contactMethodType?.id ?? '')) {
```

Comment:
this also seems funky, I don't think we should be trying to match empty string with validtypes if the contactmethodtype or the id is null/undefined.